packages/mediacenter/kodi/patches/aarch64/kodi-0002-AESinkALSA-force-default-device-for-non-passthrough.patch

```diff
@@ -11,14 +11,14 @@ diff --git a/xbmc/cores/AudioEngine/Sinks/AESinkALSA.cpp b/xbmc/cores/AudioEngin
 index 6a9066b..4d7f85d 100644
 --- a/xbmc/cores/AudioEngine/Sinks/AESinkALSA.cpp
 +++ b/xbmc/cores/AudioEngine/Sinks/AESinkALSA.cpp
-@@ -535,9 +535,8 @@ bool CAESinkALSA::Initialize(AEAudioFormat &format, std::string &device)
+@@ -535,8 +535,8 @@ bool CAESinkALSA::Initialize(AEAudioFormat &format, std::string &device)
      m_passthrough   = false;
    }
  #if defined(HAS_LIBAMCODEC)
 -  if (aml_present())
 +  if (!m_passthrough && device.find("hdmi:CARD=AMLM8AUDIO") != std::string::npos)
    {
 -    aml_set_audio_passthrough(m_passthrough);
-     device = "default";
++    device = "default";
```

Comment:
This is wrong and will stop USB-sound from working again, `device = "default"` was removed from upstream and from the previous patch in LE.